### PR TITLE
Fixes and Rev4 updates for aide controls

### PR DESCRIPTION
**Pull request by samcornwell**
These are all the changes we have made in our repo at https://github.com/djhaynes/inspec-profile-disa_stig-el7 relating to aide controls. This includes updates of these controls to RHEL7 STIG Rev4, as well as various fixes.

### CODE REVIEW

File: controls/V-71973.rb [hunk 1]
```diff
@@ -1,54 +1,34 @@
...
-uri: http://iase.disa.mil
------------------
-=end
-
-file_integ_tool = attribute(
-  'file_integ_tool',
-  default: 'aide',
-  description: "Tool used to determine file integrity"
-)
+FILE_INTEGRITY_TOOL = attribute('file_integrity_tool', default: 'aide',
```
Comment (trevor-vaughan) [inline controls/V-71973.rb]: No globals

File: controls/V-71973.rb [hunk 2]
```diff
@@ -60,39 +40,56 @@
...
-  describe package(file_integ_tool) do
+0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"aide integrity check run for <system name>\" root@sysname.mil"
+  tag "fix_id": "F-78325r1_fix"
+  describe package(FILE_INTEGRITY_TOOL) do
     it { should be_installed }
   end
-  describe.one do
-    describe file("/etc/cron.daily/#{file_integ_tool}") do
-      it { should exist }
+  if FILE_INTEGRITY_INTERVAL == 'monthly'
```
Comment (trevor-vaughan) [inline controls/V-71973.rb]: This can also be in root's crontab and/or `/etc/cron.d` so those should also be checked

File: controls/V-71975.rb [hunk 3]
```diff
@@ -31,80 +12,79 @@
...
-  describe package(file_integ_tool) do
+0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"$HOSTNAME - Daily aide integrity check run\" root@sysname.mil"
+  tag "fix_id": "F-78327r1_fix"
+  describe package(FILE_INTEGRITY_TOOL) do
     it { should be_installed }
   end
   describe.one do
-    describe file("cat /etc/cron.daily/#{file_integ_tool}") do
-      its('content') { should match /\/bin\/mail/ }
+    describe file("/etc/cron.daily/#{FILE_INTEGRITY_TOOL}") do
```
Comment (trevor-vaughan) [inline controls/V-71975.rb]: This could also be in root's crontab or `/etc/cron.d` so those should also be checked.

**Comment by samcornwell** [comment]
@trevor-vaughan I addressed all of your comments, should be good to go

**Pull request merged**